### PR TITLE
chore: update ito abi

### PR DESCRIPTION
**Pull request by hanszhoou**
## Description
Update ITO withdraw status by chain.

Closes # (NO_ISSUE)

## Type of change

- [ ] Documentation
- [ ] Code refactoring (Restructuring existing code w/o changing its observable behavior)
- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (a fix or feature that would make something no longer possible to do/require old user must upgrade their Mask Network to this new version)

## Checklist

- [ ] My code follows the style guidelines of this project.
- [ ] I have performed a self-review of my own code.
  - [ ] I have removed all in development `console.log`s
  - [ ] I have removed all commented code.
- [ ] I have commented on my code, particularly in hard-to-understand areas.
- [ ] I have read [Internationalization Guide](https://github.com/DimensionDev/Maskbook/blob/develop/docs/i18n-guide.md) and moved text fields to the i18n JSON file.

If this PR depends on external APIs:

- [ ] I have configured those APIs with CORS headers to let extension requests get passed. 
  - chrome extension: `chrome-extension://[id]`
  - firefox extension: `moz-extension://[id]`
- [ ] I have delegated all web requests to the background service via the internal RPC bridge.

### CODE REVIEW

File: packages/mask/src/plugins/ITO/SNSAdaptor/hooks/useAvailabilityComputed.ts [hunk 1]
```diff
@@ -56,7 +56,7 @@ export function useAvailabilityComputed(payload: JSON_PayloadInMask | JSON_Paylo
             unlockTime,
             qualificationAddress,
             hasLockTime,
-            isUnlocked: availability.unlocked,
+            isUnlocked: unlockTime < Date.now(),
```
Comment (septs) [inline packages/mask/src/plugins/ITO/SNSAdaptor/hooks/useAvailabilityComputed.ts]: https://date-fns.org/v2.28.0/docs/isAfter

File: packages/mask/src/plugins/ITO/SNSAdaptor/ITO.tsx [hunk 2]
```diff
@@ -315,9 +315,8 @@ export function ITO(props: ITO_Props) {
         .toString()
     const canWithdraw = useMemo(
         () =>
+            !availability?.destructed &&
             isAccountSeller &&
-            !tradeInfo?.destructInfo &&
-            !loadingTradeInfo &&
             !availability?.exchanged_tokens.every((t) => t === '0') &&
```
Comment (septs) [inline packages/mask/src/plugins/ITO/SNSAdaptor/ITO.tsx]:
```suggestion
            !availability?.exchanged_tokens.every(isZero) &&
```

File: packages/mask/src/plugins/ITO/SNSAdaptor/hooks/useClaimablePools.ts [hunk 3]
```diff
@@ -81,12 +81,13 @@ export function useClaimablePools(chainId: ChainId, isMainnetOld = false) {
...
             )
             .map((raw) => {
+                const unlockTime = Number(raw.availability.unlock_time) * 1000
                 return {
                     pids: [raw.pid],
                     amount: Number(raw.availability.swapped),
                     token: raw.token,
-                    isClaimable: raw.availability.unlocked,
-                    unlockTime: new Date(Number(raw.availability.unlock_time) * 1000),
+                    isClaimable: unlockTime < Date.now(),
```
Comment (septs) [inline packages/mask/src/plugins/ITO/SNSAdaptor/hooks/useClaimablePools.ts]: https://date-fns.org/v2.28.0/docs/isAfter

**Comment by orange187** [comment]
Checked

**Review by guanbinrui — APPROVED** [review]

**Pull request merged**